sdk/storage/storage-blob/src/BrowserPolicyFactory.ts

```diff
@@ -3,6 +3,7 @@
 
 import { RequestPolicy, RequestPolicyFactory, RequestPolicyOptions } from "@azure/core-http";
 import { BrowserPolicy } from "./policies/BrowserPolicy";
+export { BrowserPolicy };
```

Comment:
Why is this being exported? 
From what I can see, the `BrowserPolicy` is used only in the `BrowserPolicyFactory` which in turn is internal to the library and not exposed to the user

Reply:
There may be a mistake then, because some of the policy factories __are__ currently being exported. Are the factories meant to be exported?

I exported this and the `RetryPolicy` because the factories referenced them and were being exported for users.

Reply:
That's interesting. @bterlson are we ok with these policies being exported?

Reply:
If we are exporting it, then we should probably add a "Storage" prefix to it, right?
Especially for the retry policy.

Reply:
cc @bterlson

Reply:
BrowserPolicy should be exported since it's a policy specific to Storage and users might want to compose their own pipeline with BrowserPolicy in it.

StorageBrowserPolicy makes sense (and so too does a general guideline requiring some prefix for policies so it's clear their service-specific), but considering that they can only import this from the storage package itself, it's not super important I think.

Reply:
Renamed to `StorageBlobBrowserPolicy` since the policies aren't shared across storage packages.

Reply:
The rename is part of #5862